readme.md

```diff
@@ -1,7 +1,21 @@
 # Readme.md
 
-This file has notes on building the API reference
-for the Roslyn APIs.
+This repository contains documentation for the
+[Roslyn platform](https://github.com/dotnet/roslyn) and the
+[.NET Compiler Platform SDK](https://github.com/dotnet/roslyn-sdk). The
+Roslyn platform contains the compiler and is part of every .NET installation.
+The Compiler Platform SDK provides tools and templates for programmers working
+with the Roslyn platform. The .NET Compiler
+Platform SDK is part of the overall .NET developer documentation.
+The .NET content team tracks all work for .NET developer documentation
+on the [.NET Docs](https://github.com/dotnet/docs) repository. If you find
+any issues, or have suggestions on how to improve the .NET Compiler platform
+SDK documentation, create them on the [docs repository](https://github.com/dotnet/docs/issues).
+That repository contains the source for conceptual documentation, tutorials and more,
+including all the documentation on the .NET Compiler Platform SDK.
+
+This remainder of this file has notes which NuGet packages make up the
```

Comment:
This remainder -> The remainder

has notes which reads a bit funny. Maybe add the word about? has notes about which